src/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_executable(asteroid-calendar main.cpp resources.qrc)
```

Comment:
Can you change this line also to `add_library`?

Reply:
I already did that but seems I never pushed it :see_no_evil:

Reply:
Ah, that explains it, thanks! :smile: